descriptors/ModuleDescriptor-template.json

```diff
@@ -23,6 +23,7 @@
             "users.item.get",
             "users.collection.get",
             "users.item.post",
+            "users.item.put",
```

Comment:
We are updating the requester using system user, so this permission needs to be added to system user's permissions